src/main/steps/applicant1/hub-page/joint/latest-update-content/final-order-complete.njk

```diff
@@ -0,0 +1,4 @@
+{% if isFinalOrderComplete %}
+  <p class="govuk-body">{{ finalOrderComplete.line1 }}</p>
+  <p class="govuk-body">{{ finalOrderComplete.line2 | safe }}</p>
```

Comment:
this is where you'll have the a tag, you can see other places where we do this.

Reply:
Moved `<a></a>`  to .njk files